src/database/migrations/20230612024757-create-entity-types.js

```diff
@@ -30,6 +27,9 @@ module.exports = {
 			allow_filtering: {
 				type: Sequelize.BOOLEAN,
 			},
+			data_type: {
```

Comment:
This is about if entities are of type integer, string, float etc right?

Reply:
Resolved [here](https://github.com/ELEVATE-Project/mentoring/pull/258/files#diff-b4a89c05413a124cef2b14c7268289a97fd1ea31542977c7e7edab2c75b1d23e).